src/main/kotlin/org/rust/ide/console/RunRustConsoleAction.kt

```diff
@@ -0,0 +1,22 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.ide.console
+
+import com.intellij.notification.NotificationType
+import com.intellij.openapi.actionSystem.AnActionEvent
+import com.intellij.openapi.application.TransactionGuard
+import com.intellij.openapi.project.DumbAwareAction
+import org.rust.ide.notifications.showBalloonWithoutProject
+
+class RunRustConsoleAction : DumbAwareAction() {
```

Comment:
Please disable this action in case of a non-Rust project:

```Kotlin
    override fun update(e: AnActionEvent) {
        e.presentation.isEnabled = e.project?.hasCargoProject == true
    }
```